static/src/javascripts/projects/commercial/modules/header-bidding/prebid/bid-config.ts

```diff
@@ -468,27 +468,27 @@ const biddersSwitchedOn = (allBidders: PrebidBidder[]): PrebidBidder[] => {
 };
 
 const currentBidders = (slotSizes: HeaderBiddingSize[]): PrebidBidder[] => {
-	const otherBidders: PrebidBidder[] = [
-		...(inPbTestOr(shouldIncludeCriteo()) ? [criteoBidder] : []),
-		...(inPbTestOr(shouldIncludeSmart()) ? [smartBidder] : []),
-		...(inPbTestOr(shouldIncludeSonobi()) ? [sonobiBidder] : []),
-		...(inPbTestOr(shouldIncludeTrustX()) ? [trustXBidder] : []),
-		...(inPbTestOr(shouldIncludeTripleLift()) ? [tripleLiftBidder] : []),
-		...(inPbTestOr(shouldIncludeAppNexus()) ? [appNexusBidder] : []),
-		...(inPbTestOr(shouldIncludeImproveDigital())
-			? [improveDigitalBidder]
-			: []),
-		...(inPbTestOr(shouldIncludeImproveDigitalSkin())
-			? [improveDigitalSkinBidder]
-			: []),
-		...(inPbTestOr(shouldIncludeXaxis()) ? [xaxisBidder] : []),
-		pubmaticBidder,
-		...(inPbTestOr(shouldIncludeAdYouLike(slotSizes))
-			? [adYouLikeBidder]
-			: []),
-		...(inPbTestOr(shouldUseOzoneAdaptor()) ? [ozoneClientSideBidder] : []),
-		...(shouldIncludeOpenx() ? [openxClientSideBidder] : []),
```

Comment:
Note that this line is actually missing the `inPbTestOr(...)` call, which I assume isn't intentional? After this change this call will be present for all bidders.